webapp/pages/login.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <LoginForm></LoginForm>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style>
+</style>
```

Comment:
as already mentioned above ⭐ ⭐ For a login feature in your webapp including a Vue component and its software tests.